packages/kolibri-tools/lib/i18n/crowdin/cli.py

```diff
@@ -216,5 +203,27 @@ def dispatch_command():
 
     glossary_path_help = "The relative location of the Glossary file you want to pass to the given command"
     parser.add_argument("--glossary-path", "-g", type=str, help=glossary_path_help)
-    PARSED_ARGS = vars(parser.parse_args())
+
+    # Load the config file, update it with the parsed args (passed args overload config)
+    config = read_config_file("../../")
```

Comment:
We should read from `os.cwd()` for the config file, otherwise this is very sensitive to where the package is installed for actually discovering the config (noting that if no `path` argument is passed, this is what `read_config_file` does by default anyway).

Reply:
The issue I'm having here I think is that the way the package is configured it's going to be run from wherever it is in the system because it isn't a PyPi package -- so the i18n Make command I had was like `cd packages/kolibri-tools && python -m lib.i18n.crowdin.cli ...` 

Maybe I'll just need to add a `config_path` option and put it on the user (us) to tell us where it is. I'll think on how to document this a bit better if that's how we do it

Reply:
The reasoning was that `node_modules/kolibri-tools/../../<Studio|KDP|etc>` and `packages/kolibri-tools/../../<Kolibri>` make this work out well enough but obviously is not ideal to rely on directory structure for a tool to work

Reply:
The other option would be to add an entry point in the package.json that invokes the python code, so that it can be run without the `cd`ing?

I am leaning slightly towards the idea of just publishing this as a little sidecar Python library, tbh, might make it easier.